### DISCUSSION TITLE
How can I get response body?

### DISCUSSION THREAD

**Discussion by MeMpy** [kaliiiiiiiiii/Selenium-Driverless#110] [Q&A] [answered]
Hello, thanks for your powerful driver. 
I am wondering if there is any method to get the response body of all the requests done within a page. Right now I am using Python with sync approach so sync solution are much appreciated but if the only way is to use async I will adapt my code to it.

With a regular chrome driver in the past I used to get the performance logs then from it find the request i am interested of and then get the request id and use it in the cdp dev tool to retrive `Network.responseBody`. But with your driver I cannot get performance logs anymore ( for what I understood ).

**Comment by ttlns** [comment, marked as answer]
Does https://github.com/kaliiiiiiiiii/Selenium-Driverless#use-events (request interception) answer you that?

**Reply by MeMpy** [reply]
It did!! Thanks!
